fhir-server-test/src/test/java/org/linuxforhealth/fhir/server/test/SortingTest.java

```diff
@@ -1126,4 +1170,53 @@ public void testSearchResultsOperationOutcomeWarning() throws Exception {
         }    
         assertEquals(warningMessage, "Pages have shifted; check pages for changed results.");
     }
+    
+    /**
+     * validate whether the search result has no 'Pages have shifted' warning. Fail the test if the search result has 'Pages have shifted' warning.
+     * @param bundle the search response bundle
+     */
+    private void validateNoSearchResponseWarningExists(Bundle bundle) {
+        String warningMessage = null;
+        for (Bundle.Entry entry : bundle.getEntry()) {
+            if(entry.getSearch().getMode().getValue().equals("outcome")) {
```

Comment:
slightly safer to reverse this (`"outcome".equals(entry.getSearch().getMode().getValue()`)
but maybe even better to compare the enum values:
```suggestion
            if (entry.getSearch().getMode().getValueAsEnum() == SearchEntryMode.OUTCOME)) {
```

Reply:
on the other hand, since its just a test, I think its fine to leave as-is for now

Reply:
This has been fixed as suggested.